test/common/markdown/markdown.tests.ps1

```diff
@@ -102,15 +102,18 @@ Describe 'Common Tests - Validate Markdown Files' -Tag 'CI' {
 
         $mdIssuesPath | should exist
 
+        [string] $markdownErrors = ""
         Get-Content -Path $mdIssuesPath | ForEach-Object -Process {
             if ([string]::IsNullOrEmpty($_) -eq $false -and $_ -ne '--EMPTY--')
             {
                 Write-Warning -Message $_
```

Comment:
Yeah, it's redundant now.  I'm also going to remove the count as that code is also redundant with the check.